etl/steps/data/garden/AI/2023-06-08/yougov_job_automation.meta.yml

```diff
@@ -0,0 +1,209 @@
+dataset:
+  title: How worried are Americans about being automated out of a job? (YouGov, 2023)
+  description: Biannual tracker on how worried are Americans about being automated out of a job? The provided results are weighted to ensure they are representative of the United States population across various demographic factors. The weighting is based on factors such as age, gender, race, education, region, political party affiliation, and income level. This weighting process helps to account for the diversity within the population and ensures that the results reflect a broader representation of the United States as a whole. Next update in July. Each wave comprises responses from 460 to 495 working US adults.
```

Comment:
As I mentioned in another yaml file, you can avoid repeating long definitions by using yaml anchors `&` and aliases `*`.